llm_github_models.py

```diff
@@ -35,67 +43,90 @@
-    ("Meta-Llama-3-8B-Instruct", True, False, False, ["text"], ["text"]),
-    ("Meta-Llama-3.1-405B-Instruct", True, False, False, ["text"], ["text"]),
-    ("Meta-Llama-3.1-70B-Instruct", True, False, False, ["text"], ["text"]),
-    ("Meta-Llama-3.1-8B-Instruct", True, False, False, ["text"], ["text"]),
-    ("Ministral-3B", True, False, False, ["text"], ["text"]),
-    ("Mistral-Large-2411", True, False, False, ["text"], ["text"]),
-    ("Mistral-Nemo", True, False, False, ["text"], ["text"]),
-    ("Mistral-large", True, False, False, ["text"], ["text"]),
-    ("Mistral-large-2407", True, False, False, ["text"], ["text"]),
-    ("Mistral-small", True, False, False, ["text"], ["text"]),
-    ("Phi-3-medium-128k-instruct", True, False, False, ["text"], ["text"]),
-    ("Phi-3-medium-4k-instruct", True, False, False, ["text"], ["text"]),
-    ("Phi-3-mini-128k-instruct", True, False, False, ["text"], ["text"]),
-    ("Phi-3-mini-4k-instruct", True, False, False, ["text"], ["text"]),
-    ("Phi-3-small-128k-instruct", True, False, False, ["text"], ["text"]),
-    ("Phi-3-small-8k-instruct", True, False, False, ["text"], ["text"]),
-    ("Phi-3.5-MoE-instruct", True, False, False, ["text"], ["text"]),
-    ("Phi-3.5-mini-instruct", True, False, False, ["text"], ["text"]),
-    ("Phi-3.5-vision-instruct", True, False, False, ["text", "image"], None),
-    ("Phi-4", True, False, False, ["text"], ["text"]),
-    ("Phi-4-mini-instruct", True, False, False, ["text"], ["text"]),
-    ("Phi-4-mini-reasoning", True, False, False, ["text"], ["text"]),
-    ("Phi-4-multimodal-instruct", True, False, False, ["audio", "image", "text"], ["text"]),
-    ("Phi-4-reasoning", True, False, False, ["text"], ["text"]),
-    ("cohere-command-a", True, False, False, ["text"], ["text"]),
-    ("gpt-4.1", True, True, True, ["text", "image"], ["text"]),
-    ("gpt-4.1-mini", True, True, True, ["text", "image"], ["text"]),
-    ("gpt-4.1-nano", True, True, True, ["text", "image"], ["text"]),
-    ("gpt-4o", True, True, True, ["text", "image", "audio"], ["text"]),
-    ("gpt-4o-mini", True, True, True, ["text", "image", "audio"], ["text"]),
-    ("grok-3", True, False, False, ["text"], ["text"]),
-    ("grok-3-mini", True, False, False, ["text"], ["text"]),
-    ("jais-30b-chat", True, False, False, ["text"], ["text"]),
-    ("mistral-medium-2505", True, False, False, ["text", "image"], ["text"]),
-    ("mistral-small-2503", True, False, False, ["text", "image"], ["text"]),
-    ("o1", False, True, False, ["text", "image"], ["text"]),
-    ("o1-mini", False, False, False, ["text"], ["text"]),
-    ("o1-preview", False, False, False, ["text"], ["text"]),
-    ("o3", True, False, True, ["text", "image"], ["text"]),
-    ("o3-mini", False, True, False, ["text"], ["text"]),
-    ("o4-mini", True, False, True, ["text", "image"], ["text"]),
+    ("AI21-Jamba-1.5-Large", True, False, False, False, ["text"], ["text"]),
```

Comment:
These tuples are getting out of control 😆

Reply:
It would be great if the GitHub Models API had an endpoint that returned all of these details!